cppcache/src/SystemProperties.cpp

```diff
@@ -169,7 +169,8 @@ SystemProperties::SystemProperties(
       m_sslTrustStore(DefaultSslTrustStore),
       m_sslKeystorePassword(DefaultSslKeystorePassword),
       m_conflateEvents(DefaultConflateEvents),
-      m_threadPoolSize(DefaultThreadPoolSize),
+      // m_threadPoolSize(DefaultThreadPoolSize),
+      m_threadPoolSize(16),
```

Comment:
Looks like an unintentional change!

Reply:
Good catch. Yup that was for investigating impact of threadLocals on TSSDataOutput buffers. Reverted.